content/en/guides/get-started/installation.md

```diff
@@ -8,29 +8,71 @@ csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/
 
 ## Prerequisites
 
-Here, you will find information on setting up and running a Nuxt.js project in 4 steps.
+- [node](https://nodejs.org/en/download/) - at least v8.9.0 _We recommend you have the latest version installed._
+- Text editor: Use whatever you like, but we recommend [VSCode](https://code.visualstudio.com/).
+- Use whatever you like, but we recommend using VSCode's [integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal).
 
-<base-alert type="info">
+## Using create-nuxt-app
 
-Another way to get started with Nuxt.js is to use [CodeSandbox](https://template.nuxtjs.org) which is a great way for quickly playing around with Nuxt.js and/or sharing your code with other people.
+To get started quickly you can use the [create-nuxt-app](https://github.com/nuxt/create-nuxt-app).
 
-</base-alert>
+Make sure you have npx installed (npx is shipped by default since NPM 5.2.0) or npm v6.1 or yarn.
+
+<code-group>
+  <code-block label="Yarn" active>
+
+```bash
+yarn create nuxt-app <project-name>
+```
+
+  </code-block>
+  <code-block label="NPX">
+
+```bash
+npx create-nuxt-app <project-name>
+```
+
+  </code-block>
+    <code-block label="NPM">
+
+```bash
+npm init nuxt-app <project-name>
+```
 
-### Node
+  </code-block>
 
-[node](https://nodejs.org/en/download/) - at least v8.9.0
+</code-group>
 
-_We recommend you have the latest version installed._
+It will ask you some questions (name, Nuxt options, UI framework, TypeScript, linter, testing framework, etc.), when answered, it will install all the dependencies. The next step is to navigate to the project folder and launch it:
```

Comment:
Maybe it'd be good to roughly go through each option and explain what they are there for without going too deep?

Reply:
we removed this as it was too difficult to update as it keeps changing and it is clear enough in the create nuxt app which it links to and all info is there

Reply:
> we removed this as it was too difficult to update as it keeps changing 

Isn't this rather a bad sign? Rule of thumb: If it is hard to document, it is difficult for the user as well.

> and it is clear enough in the create nuxt app which it links to and all info is there

Maybe it's worth adding a link again, e.g. "Find out more about each question step here"?

Reply:
it's more about not having duplicate content to maintain. At the end of the day people just want to run create nuxt app and get an app up and running. I did create a video on it which was 15 mins long and goes over what each one is. This is too much to document as it will scare people away. We could always add the video into the documentation or the link like you suggested - Find out more......

Reply:
As long as there is some "Find out more about all the options" link I'm fine with it. Reducing redundant content is good as long as the content is still accessible for the read/user/... 👍

Reply:
added